### PR TITLE
Format test

### PR DESCRIPTION
**Pull request by mmatera**
This is a cleaner and reduced version of #184, and one of the parts of #303. 

<a href="https://gitpod.io/#https://github.com/Mathics3/mathics-core/pull/303"></a>

### CODE REVIEW

**Comment by rocky** [comment]
@mmatera  I am going to be making a lot of changes to master over the weekend. If there changes, hopefully of the smaller sort or of PRs from before that should have been merged,  it would be good to get those in sooner.

File: mathics/builtin/box/inout.py [hunk 1]
```diff
@@ -387,8 +382,6 @@ def apply(self, a, b, evaluation, options):
     def init(self, a, b, options):
         self.box_options = options.copy()
         if not (isinstance(a, BoxConstruct) and isinstance(b, BoxConstruct)):
-            print("a=", a)
```
Comment (rocky) [inline mathics/builtin/box/inout.py:390]: You know if you want to put in print statements and see that actions CI works, you could do this in your own private branch and set that up to run actions. 

Thanks for your consideration.

Comment (mmatera) [reply mathics/builtin/box/inout.py:390]: This is why I removed them.

**Comment by mmatera** [comment]
> @mmatera I am going to be making a lot of changes to master over the weekend. If there changes, hopefully of the smaller sort or of PRs from before that should have been merged, it would be good to get those in sooner.

I am on it. I would like to merge this branch and the related ones, but first I want to implement the corrections in the comments. I hope to finish during the morning.

File: mathics/builtin/box/inout.py (hunk 1 above)
Comment (rocky) [reply mathics/builtin/box/inout.py:390]: I guess what I am saying is that this never should have appeared in a PR, even a draft of a PR.

**Comment by mmatera** [comment]
OK, let's merge this and iterate.

**Pull request merged**